desk/app/groups.hoon

```diff
@@ -1012,6 +1017,32 @@
       (emit %give %kick ~ ~)
     go-core
   ::
+  ++  go-clone
+    =.  cor
+      %-  emil
+      ^-  (list card)
+      %~  tap  in
+      ^-  (set card)
+      %-  %~  run  in
+      ^-  (set nest:g)
+      %~  key  by  channels.group
+      |=  =nest:g
+      ?+  p.nest  !!
+          %chat
+        =/  =log:c  .^(log:c %gx (welp (channel-scry nest) /logs/noun))
+        =/  =clone:cl  [q.flag q.q.nest [%chat log]]
+        [%give %fact ~[/clone/(scot %p p.q.nest)/[q.q.nest]] group-clone+!>(clone)]
```

Comment:
We probably want to only give these facts to the ship that just opened the subscription, instead of all ships that still have this subscription open.

Along the same vein, we probably want to `%kick` the incoming subscription after we give it these facts.